hack/validate/golangci-lint

```diff
@@ -17,6 +17,8 @@ elif ${PKG_CONFIG} 'libsystemd-journal' 2> /dev/null ; then
 	DOCKER_BUILDTAGS+=" journald journald_compat"
 fi
 
+echo -e "\n\033[0;36mINFO\033[0m Start validation with golang-ci-lint"
```

Comment:
Does this script ever get piped to a file or similar where these control codes will just be binary noise, or is it pretty exclusively terminal/Jenkins?

Reply:
This should only be in CI (afaik)